spec/src/main/asciidoc/ch03-entity-operations.adoc

```diff
@@ -1162,36 +1162,42 @@ public interface EntityManager extends AutoCloseable {
     public <T> List<EntityGraph<? super T>> getEntityGraphs(Class<T> entityClass);
 
     /**
-     * Execute the given action using the {@link java.sql.Connection} underlying
-     * this {@code EntityManager}. If this {@code EntityManager} is associated
-     * with a transaction, the action is executed in the context of the transaction.
-     * The given action should close any JDBC resources it creates, but should not
-     * close the connection itself, nor commit or roll back the transaction. If the
-     * given action throws an exception, the persistence provider must mark the
+     * Execute the given action using the database connection underlying this
+     * {@code EntityManager}. Usually, the connection is a JDBC connection, but a
+     * provider might support some other native connection type, and is not required
+     * to support {@code java.sql.Connection}. If this {@code EntityManager} is
+     * associated with a transaction, the action is executed in the context of the
+     * transaction. The given action should close any JDBC resources it creates, but
```

Comment:
_any resources it creates_ ? likely obvious, but...

Reply:
Yeah of course, thanks, fixed now.